source/extensions/tracers/zipkin/span_context_extractor.cc

```diff
@@ -197,17 +198,19 @@ SpanContextExtractor::extractSpanContextFromB3SingleFormat(bool is_sampled) {
         throw ExtractorException(fmt::format("Invalid input: invalid sampling flag {}", b3[pos]));
       }
       pos++; // consume the sampled status
+    } else {
+      throw ExtractorException("Invalid input: truncated");
     }
 
     if (b3.length() > pos) {
+      std::cout << "pos:" << pos << "len: " << b3.length() << std::endl;
```

Comment:
delete?

Reply:
Used for debugging code, forgot to delete